accounting/src/creditcommons/routes.ts

```diff
@@ -19,6 +38,16 @@ import {
 export function getRoutes(controller: SharedController) {
   const router = Router()
 
+  /**
+   * Configure the trunkward CC node. Requires admin.
+   */
+  router.post('/:code/creditCommonsNodes', userAuth(Scope.Accounting), checkExact(CreditCommonsValidators.isGraft()),
+    currencyInputHandler(controller, async (currencyController, ctx, data: CreditCommonsNode) => {
+      console.log(data)
```

Comment:
If you want to leave these logs here, you can also use 

```
import { logger} from src/utils/logger
logger.debug(data)
```
Also, when developing, you can use the tool `pino-pretty` so the logs are readable. The `pnpm dev` already uses it, but the tests scripts don't.

Mmm, only think that remains is a env var for setting the log level to debug, but otherwise you can have a single line of code with `logger.level = 'debug'` to enable/disable all at once.

Reply:
thanks!